tool/mode_creator.html

```diff
@@ -59,6 +59,7 @@
     <select id="modeEl" size="1"></select>
     <input type="button" value="&#10227;" title="sync" id="syncToMode"></select>
     <span id="tablist"></span>
+    <input type="button" value="Upload" id="uploadToServer1"></select>
```

Comment:
> Ctrl-S to work unreliably. 

this can happen only if editor is not focused

> So if you insist on removing the buttons, you will have to do so yourself after pulling. I can't prevent that. But I'm not willing to give them up

:)
Ok, i've kept them, but made some modifications in 0054bb87946e27d534b49fb1cdf1acf1b261a888